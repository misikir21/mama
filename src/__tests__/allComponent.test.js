import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../route/home';
import CalculatorPage from '../route/calculator';
import Quote from '../route/quote';

describe('Test the all commponents', () => {
  it('The HomePage is rendered accurately! ', () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });

  test('The CalculatorPage is rendered accurately!', async () => {
    const { container } = render(<CalculatorPage />);
    expect(container).toMatchSnapshot();
  });

  it('The QuotePage is rendered accurately!', () => {
    const { container } = render(<Quote />);
    expect(container).toMatchSnapshot();
  });
});
