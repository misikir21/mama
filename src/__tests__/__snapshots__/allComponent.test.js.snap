// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the all commponents The CalculatorPage is rendered accurately! 1`] = `
<div>
  <div
    class="wrapper"
  >
    <h2
      class="title"
    >
      Let\`s do some math!
    </h2>
    <div>
      <div
        class="cal-contianer"
      >
        <div
          class="diplay"
        >
          0
        </div>
        <div
          class="but-contianers"
        >
          <button
            class="btn"
            id="AC"
            type="button"
          >
            AC
          </button>
          <button
            class="btn"
            id="+/-"
            type="button"
          >
            +/-
          </button>
          <button
            class="btn"
            id="%"
            type="button"
          >
            %
          </button>
          <button
            class="btn"
            id="รท"
            type="button"
          >
            รท
          </button>
          <button
            class="btn"
            id="7"
            type="button"
          >
            7
          </button>
          <button
            class="btn"
            id="8"
            type="button"
          >
            8
          </button>
          <button
            class="btn"
            id="9"
            type="button"
          >
            9
          </button>
          <button
            class="btn"
            id="x"
            type="button"
          >
            x
          </button>
          <button
            class="btn"
            id="4"
            type="button"
          >
            4
          </button>
          <button
            class="btn"
            id="5"
            type="button"
          >
            5
          </button>
          <button
            class="btn"
            id="6"
            type="button"
          >
            6
          </button>
          <button
            class="btn"
            id="-"
            type="button"
          >
            -
          </button>
          <button
            class="btn"
            id="1"
            type="button"
          >
            1
          </button>
          <button
            class="btn"
            id="2"
            type="button"
          >
            2
          </button>
          <button
            class="btn"
            id="3"
            type="button"
          >
            3
          </button>
          <button
            class="btn"
            id="+"
            type="button"
          >
            +
          </button>
          <button
            class="btn"
            id="0"
            type="button"
          >
            0
          </button>
          <button
            class="btn"
            id="."
            type="button"
          >
            .
          </button>
          <button
            class="btn"
            id="="
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Test the all commponents The HomePage is rendered accurately!  1`] = `
<div>
  <div
    class="wrapper"
  >
    <h2
      class="title"
    >
      Welcome to our Page
    </h2>
    <div
      class="text"
    >
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
      </p>
    </div>
    <div
      class="text"
    >
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
      </p>
    </div>
  </div>
</div>
`;

exports[`Test the all commponents The QuotePage is rendered accurately! 1`] = `
<div>
  <div
    class="qoute-contianer"
  >
    <div>
      Loading...
    </div>
    <div>
      <h2 />
      <h4 />
      <p />
    </div>
  </div>
</div>
`;
